bundles/org.openhab.binding.emotiva/src/main/java/org/openhab/binding/emotiva/internal/EmotivaCommandHelper.java

```diff
@@ -31,26 +30,26 @@
 public class EmotivaCommandHelper {
 
     public static PercentType volumeDecibelToPercentage(String volumeInDecibel) {
-        String volumeTrimmed = volumeInDecibel.replace("dB", "").trim();
-        int clampedValue = clamp(volumeTrimmed, DEFAULT_VOLUME_MIN_DECIBEL, DEFAULT_VOLUME_MAX_DECIBEL);
+        var volumeTrimmed = volumeInDecibel.replace("dB", "").trim();
```

Comment:
Why are you changing all types to var's? There is no need and i don't see a clear advantage. For me as reviewer it usually makes it harder.

Reply:
Excellent question. Really just trying to use some of the new Java lang features, do see the point about reviewing. What if we use var in places that there is an instanciation (new ...) or use of static methods, like Duration.of(), but dropping all places where you get the result from a method or other places it is not clear what the type is? Or do you think revert it all is the best?

Reply:
In my opinion `var` should only be used when the type is obvious when looking at RHS. This is in line with @lsiepel's good point about reviewing code.

This is quite strict and also excludes `Duration.of()`, because you cannot directly see if that returns a `Duration` or something else. Basically this means that I'd only use it in cases like this:
```java
var foo = new Foo();
```

because in this case it's obvious that you're getting a Foo, thus it seems redundant to specify it twice:
```java
Foo foo = new Foo();
```

Reply:
And even in that case, IMO it is ‘the same but different’. If there are long (nested) type names, I can see a small advantage  when ‘var’ reduces the clutter / multiline (thank you spotless). But yeah, not really related to this PR.

Reply:
I'm wired for consistency, so if I'm doing:
```java
var obj = new ClassForSomeReasonHavingVeryLongAndPainfulNameResultingInSpotlessIssues(parm1, parm2, parm3);
```

I would be giving the same treatment for:
```java
var obj = new Foo();
```

without making an exception here, just because it's short. And I wouldn't be doing it because of Spotless, I would do it to avoid redundancy.

At work we have strict rules for this (in C#). People most often tend to fall into the trap of assuming RHS type in situations like this:
```csharp
var logic = this.GetBusiness<ILogic>();
```

For any developer working on this system, we know we'll get an `ILogic` object from this, but since **you** don't know this system, you probably more clearly get the point why our rules are not to use `var` here. 😄

Reply:
I have pushed an update that reverts many of these changes, but keeps them where it reduces verbosity.